src/components/TracePage/TraceTimelineViewer/SpanDetail/AccordianKeyValues.js

```diff
@@ -22,16 +23,19 @@
 import React from 'react';
 
 import KeyValuesTable from './KeyValuesTable';
-import toggleEnhancer from './toggle-enhancer';
 
 import './AccordianKeyValues.css';
 
-type KeyValuesSummaryProps = {
+type AccordianKeyValuesProps = {
+  compact?: boolean,
   data: { key: string, value: any }[],
+  highContrast?: boolean,
+  isOpen: boolean,
+  label: string,
+  onToggle: () => void,
 };
 
-function KeyValuesSummary(props: KeyValuesSummaryProps) {
-  const { data } = props;
+function getKVSummary(data: { key: string, value: any }[]) {
```

Comment:
Any advantage in changing this? Why not just make this a React function component? I'm still a fan of being explicit with variable names aka `KeyValue` instead of `KV`. Although I know you have some preference of short names that are declared within close scope.

`<KVSummary data={..} />`

Reply:
Well, it's a lot more performant as a regular function which is invoked, but that's kind of minor considering these are relatively rare (requiring the details to be expanded).

I've reverted the change.